pymc/logprob/rewriting.py

```diff
@@ -484,7 +484,7 @@ def cleanup_ir(vars: Sequence[Variable]) -> None:
 
 
 def assume_measured_ir_outputs(
-    inputs: Sequence[TensorVariable], outputs: Sequence[TensorVariable]
+    inputs: Collection[TensorVariable], outputs: Sequence[TensorVariable]
```

Comment:
Why does it prefer collection?

Reply:
It was called with a `dict[TensorVariable, something]`, which is a `Collection[TensorVariable]` (but not a sequence).

The code here didn't need `inputs` to be in some order.